src/jaxsim/mujoco/loaders.py

```diff
@@ -159,13 +159,19 @@ def convert(
         considered_joints: list[str] | None = None,
         plane_normal: tuple[float, float, float] = (0, 0, 1),
         heightmap: bool | None = None,
+        cameras: (
+            list[dict[str, str, str, str, str]] | dict[str, str, str, str, str]
+        ) = None,
```

Comment:
Shouldn't a dictionary str -> str be typed as just `dict[str, str]`?

```suggestion
        cameras: dict[str, str] | list[dict[str, str]] | None = None,
```

Reply:
Done in https://github.com/ami-iit/jaxsim/pull/133/commits/e1ee453c899cde1faba9baf9634cb189446119c0